fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskInfoBuilder.java

```diff
@@ -147,18 +161,44 @@ public AnalysisTaskInfoBuilder copy() {
                 .setCatalogName(catalogName)
                 .setDbName(dbName)
                 .setTblName(tblName)
-                .setPartitionNames(partitionNames)
+                .setColToPartitions(colToPartitions)
                 .setColName(colName)
                 .setIndexId(indexId)
                 .setJobType(jobType)
+                .setAnalysisMode(analysisMode)
                 .setAnalysisMethod(analysisMethod)
                 .setAnalysisType(analysisType)
                 .setSamplePercent(samplePercent)
                 .setSampleRows(sampleRows)
+                .setPeriodTimeInMs(periodTimeInMs)
                 .setMaxBucketNum(maxBucketNum)
                 .setMessage(message)
                 .setLastExecTimeInMs(lastExecTimeInMs)
                 .setState(state)
                 .setScheduleType(scheduleType);
     }
+
+    public static AnalysisTaskInfoBuilder getFromJobInfo(AnalysisTaskInfo info) {
```

Comment:
Quite  wired here, it named as `getFromJobInfo` however all this method done is just copy field from another `AnalysisTaskInfo`, so why just add a constructor like `public AnalysisTaskInfoBuilder(AnalysisTaskInfo info)`

Reply:
yes, you are right. AnalysisTaskInfoBuilder(AnalysisTaskInfo info) is better